xml/1930/cc271libr_001-2321.xml

```diff
@@ -754,7 +754,7 @@
-<copyrightEntry id="1717D971-6E16-1014-BA0A-C4B2DFF7CF27" regnum="A.-Foreign 4644"><author><authorName>Bandrowski, Juliusz Kaden</authorName>, <authorBirth>1885</authorBirth>&#x2013;</author> <title>&#x2026; Novellen.</title> <publisher><pubPlace>Berlin-Grunewald</pubPlace>, <pubName>Horen-verlag</pubName>, <pubDate date="1928">1928</pubDate>. </publisher><desc>208, [1] p., 1 l. 19&#x368;&#x36B;.</desc> <note>(On cover: Die polnische literatur der gegen-wart) At head of title: Julius Kaden-Bandrowski. "Autorisierte &#xFC;bertragung von A. von Guttry." "Erstes und zweites tausend." CONTENTS.&#x2014;Hier bin ich nun&#x2014;und wnrte.&#x2014;G&#xF6;tter.&#x2014;Politik.&#x2014;Der letzte namenstag</note>. &#xA9; <regDate date="1928-11-01">Nov. 1, 1928</regDate>; <copies>1c.</copies> <copyDate date="1929-02-05">Feb. 5, 1929</copyDate>; <regNum>A.-Foreign 4644</regNum>; <publisher><pubName claimant="yes">Horen-verlag, g. m. b. h.</pubName> </publisher> (<lccn>30-5815</lccn>)</copyrightEntry>
+<copyrightEntry id="1717D971-6E16-1014-BA0A-C4B2DFF7CF27" regnum="AF4644"><author><authorName>Bandrowski, Juliusz Kaden</authorName>, <authorBirth>1885</authorBirth>&#x2013;</author> <title>&#x2026; Novellen.</title> <publisher><pubPlace>Berlin-Grunewald</pubPlace>, <pubName>Horen-verlag</pubName>, <pubDate date="1928">1928</pubDate>. </publisher><desc>208, [1] p., 1 l. 19&#x368;&#x36B;.</desc> <note>(On cover: Die polnische literatur der gegen-wart) At head of title: Julius Kaden-Bandrowski. "Autorisierte &#xFC;bertragung von A. von Guttry." "Erstes und zweites tausend." CONTENTS.&#x2014;Hier bin ich nun&#x2014;und wnrte.&#x2014;G&#xF6;tter.&#x2014;Politik.&#x2014;Der letzte namenstag</note>. &#xA9; <regDate date="1928-11-01">Nov. 1, 1928</regDate>; <copies>1c.</copies> <copyDate date="1929-02-05">Feb. 5, 1929</copyDate>; <regNum>AF4644</regNum>; <publisher><pubName claimant="yes">Horen-verlag, g. m. b. h.</pubName> </publisher> (<lccn>30-5815</lccn>)</copyrightEntry>

```

Comment:
The normalization of the `regnum` attribute on the `<copyrightEntry>` is right, but we want to leave the `<regNum>` element  as printed in the volume.

Reply:
Fixed!